src/System.Data.SqlClient/src/System/Data/SqlClient/SqlException.cs

```diff
@@ -32,9 +34,30 @@ private SqlException(string message, SqlErrorCollection errorCollection, Excepti
             _clientConnectionId = conId;
         }
 
+        private SqlException(SerializationInfo si, StreamingContext sc) : base(si, sc) 
+        {
+            HResult = SqlExceptionHResult;
```

Comment:
you are dropping exception - I assume intentioally because it's "object" - another option is try/catch

Reply:
Sorry I'm not sure what you mean here.

Reply:
Looks like I commented on the wrong file. Hopefully I will find that again..